### ISSUE TITLE
What is the sampling configuration for DIV2K 0810.png in Figure 1(a)?

### ISSUE THREAD

**Issue by KwanghyunOn** [wyhuai/DDNM#17]
Hello, I hope this message finds you well.
First of all, I wanted to express my admiration for the outstanding work you have done on your repository.

I'm currently working on a project that requires the use of the _**DIV2K 0810.png**_ image shown in **Figure 1(a)** in the paper. However, I couldn't find any information about the sampling configuration used to generate this image. If it's not too much trouble, I would like to kindly request information on the sampling configuration used to generate this image.

Thank you very much for your help!

**Comment by wyhuai** [comment]
Hi, actually I use a [guided diffusion model](https://github.com/openai/guided-diffusion) pre-trained on ImageNet 512x512 (not yet implemented in this repository), with classifier guidance, T=1000, time-travel length 10, repeat 3.

**Comment by KwanghyunOn** [comment]
Hello, thank you for your response.

I have a couple of follow-up questions.
(1) What is the class label that is used to guide the image?
(2) Is the resolution of the image shown in Figure1(a) is 512x512? I ask because the original image in the DIV2K dataset appears to have a larger resolution.

Thank you in advance for your help.

**Comment by wyhuai** [comment]
Hi, for the label, you can search "squirrel" on [this link](https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a), and get 335: 'fox squirrel, eastern fox squirrel, Sciurus niger'. 

The image in Fig.1(a) is 512x512. However, it is possible to solve higher resolution using a DDNM of only 256x256 resolution. We will submit a corresponding new work to arXiv soon.

**Comment by KwanghyunOn** [comment]
Thank you so much for your kind assistance. I look forward to seeing your future work.
